### ISSUE TITLE
meshlabserver: cannot connect to X server

### ISSUE THREAD

**Issue by LF-DevJourney** [Shicoder/Meshlab-MLXScriptBatchProcessing#1]
I run it on ubuntu server, and get `meshlabserver: cannot connect to X server`. How to run the meshlabserver without X server.

**Comment by Shicoder** [comment]
Have you installed the meshlab? Or is the path correct?

**Comment by LF-DevJourney** [comment]
Yes I've installed it, If not, there will be cannot find command meshlabserver.

**Comment by ksemianov** [comment]
Try `xvfb` 
https://github.com/cnr-isti-vclab/meshlab/issues/211#issuecomment-380077392
